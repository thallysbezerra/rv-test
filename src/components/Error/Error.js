import React from 'react';
import './Error.scss';

const Error = () => (
	<div className="error">
		Sorry! <br />
		Something went wrong. <br />
		Try again later.
	</div>
);

export default Error;
