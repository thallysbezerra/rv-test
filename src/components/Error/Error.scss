@import '../../css/theme';

.error {
	color: $red;
	font-size: 28px;
	font-weight: $medium;
	margin: 150px auto 0;
	transform: translateY(-50%);
	width: 250px;
}
