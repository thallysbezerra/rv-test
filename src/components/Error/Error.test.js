import React from 'react';
import Error from './Error';

import renderer from 'react-test-renderer';

describe('Error UI', () => {
	it('<Error /> must have the snapshot version.', () => {
		const tree = renderer.create(<Error />);

		expect(tree.toJSON()).toMatchSnapshot();
	});
});
