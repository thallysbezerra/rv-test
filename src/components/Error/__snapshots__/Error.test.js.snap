// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error UI <Error /> must have the snapshot version. 1`] = `
<div
  className="error"
>
  Sorry! 
  <br />
  Something went wrong. 
  <br />
  Try again later.
</div>
`;
